README.md

```diff
@@ -2,25 +2,36 @@
 
 ## Objectives
 
-1. Identify the default application layout.
-2. Yield to view templates from a layout.
-3. Specify a custom layout in ActionController on a controller level using the `layout` macro and on the action level using the `render :layout => "custom"` option.
+1. Identify the default application layout
+2. Yield to view templates from a layout
+3. Specify a custom layout in ActionController on a controller level using the `layout` macro and on the action level using the `render :layout => "custom"` option
 4. Use :layout => false to shut off the layout and only render the view
 
-## Outline
+## The Default Layout
 
-Give them a rails app with no layout and no controllers (besides applicationcontroller). There should be working models though. Pretty much making a shopping cart.
+1. Generate a new controller called `StaticController`, by going onto the command line and typing `rails generate controller Static`
```

Comment:
They know how to make new controllers so you can just says "Make a new controlelr called `StaticController`